tools/nodejs_api/src_cpp/node_connection.h

```diff
@@ -0,0 +1,14 @@
+#include <napi.h>
+#include <iostream>
+#include "main/kuzu.h"
+
+class NodeConnection : public Napi::ObjectWrap<NodeConnection> {
```

Comment:
Connection is not taking `numThreads` argument for initialization and also missing method `setMaxNumThreadForExec`.